File: Practice Arrays/codigo_cs/20Merge two sorted arrays.cs
using System;
using System.Linq;
public class Test20
{
    public static void MergeSort()
    {
        // your code goes here
        string input = Console.ReadLine();
        int n = int.Parse(input.Split()[0]);
        int m = int.Parse(input.Split()[1]);
        int[] a = Console.ReadLine().Split().Select(int.Parse).ToArray();
        int[] b = Console.ReadLine().Split().Select(int.Parse).ToArray();
        int[] res = new int[n + m];
        int i = 0, j = 0, k = 0;

        while (i < n && j < m)
        {
            if (a[i] <= b[j])
            {
                res[k++] = a[i++];
            }
            else
            {
                res[k++] = b[j++];
            }
        }

        while (i < n)
        {
            res[k++] = a[i++];
        }

        while (j < m)
        {
            res[k++] = b[j++];
        }

        Console.WriteLine(string.Join(" ", res));

    }
}

File: Practice Arrays/20Merge two sorted arrays.py
n,m=map(int,input().split())
a=list(map(int,input().split()))
b=list(map(int,input().split()))

res=[]
i,j=0,0
while i<n and j<m:
    if a[i]<=b[j]:
        res.append(a[i])
        i+=1
    else:
        res.append(b[j])
        j+=1
while i<n:
    res.append(a[i])
    i+=1
while j<m:
    res.append(b[j])
    j+=1

print(res)